Error 2: TestProducts.test_serialize_a_product

**As a member of the products team** ...
**I need to** fix the errors in pytest
**So that** our code runs


**Assumptions:**
* Run pytest in vagrant

**Acceptence Criteria:**
...

Error no longer shows up
...

self = <tests.test_products.TestProducts testMethod=test_serialize_a_product>

    def test_serialize_a_product(self):
        """ Test serialization of a Product """
        product = Products(name="Television", category="Electronics", available=True)
        data = product.serialize()
        self.assertNotEqual(data, None)
        self.assertIn('id', data)
        self.assertEqual(data['id'], None)
        self.assertIn('name', data)
        self.assertEqual(data['name'], "Television")
        self.assertIn('category', data)
        self.assertEqual(data['category'], "Electronics")
        self.assertIn('available', data)
>       self.assertEqual(data['available'], False)
E       AssertionError: True != False

tests/test_products.py:114: AssertionError

--- a/tests/test_products.py
+++ b/tests/test_products.py
@@ -111,7 +111,7 @@ class TestProducts(unittest.TestCase):
         self.assertIn('category', data)
         self.assertEqual(data['category'], "Electronics")
         self.assertIn('available', data)
-        self.assertEqual(data['available'], False)
+        self.assertEqual(data['available'], True)
 
     def test_deserialize_a_product(self):
         """ Test deserialization of a product """

